src/app/user/user.ts

```diff
@@ -1,4 +1,14 @@
 export class User {
+  id: string;
   fullName: string;
   imageURL: string;
 }
+
```

Comment:
Can we have a single User? We can update User class to hold the NewUser structure.